config/default.yml

```diff
@@ -80,3 +80,6 @@ MissingEnableComment:
 
 ParserBlockingJavaScript:
   enabled: true
+
+AssetSizeJavaScript:
+  enabled: true
```

Comment:
Might not want enabled by default so we can push this without forcing the 10KB rule just now.